stdlib/Test/src/Test.jl

```diff
@@ -1722,11 +1722,11 @@ Int64
 
 julia> @code_warntype f(2)
 MethodInstance for f(::Int64)
-  from f(a) @ Main none:1
```

Comment:
I don't think this will pass the docstring anymore.

Reply:
I'll revert this then